yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/YamlTestExtension.java

```diff
@@ -102,14 +105,17 @@ public Stream<TestTemplateInvocationContext> provideTestTemplateInvocationContex
      * method and a {@link YamlTestConfig}).
      */
     private static class Context implements TestTemplateInvocationContext {
+        @Nonnull
         private final YamlTestConfig config;
-        private final Set<Class<? extends YamlTestConfig>> excludedConfigs;
+        @Nonnull
         private final String excludedReason;
+        @Nullable
+        private final YamlTestConfigExclusions exclusion;
```

Comment:
This allows for a single exclusion (was a collection before). Do you think this is enough?

Reply:
This is actually covered by a previous comment... Ignore.

Reply:
Yes, it only allows a single one. I started to add the ability to repeat the annotation, but none of the tests require two reasons for excluding, so I removed that, which keeps the code simpler. If we ever have a need it's not hard to add.